recipes/genomic_regions/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "genomic_regions" %}
+{% set version = "0.0.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5cf4d27973ef0635af4c03ac8ec4a4ec9ffaf3b5d8ff9dbc216fd4926806f0fa
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - future
+    - intervaltree
+    - numpy
+    - pandas
+    - pybedtools >=0.8.0
+    - pybigwig
+    - pysam
+    - python >=3.6
+
+test:
+  imports:
+    - genomic_regions
+
```

The package includes a script. Is that also a user-facing interface? If so, you might want to add it to the tests like so:
```suggestion
  commands:
    - convert-regions --help

```

---

Thanks you for the pointer! The main reason to add this recipe is so that I can also add another recipe that depends on the genomic_regions Python API. So it is currently not necessary to test the convert-regions script.